test/image_test.py

```diff
@@ -25,7 +24,7 @@ def test_magic(f, magic_hexes):
     if len(data) != len(magic_hexes):
         return 0
     for i, magic_hex in enumerate(magic_hexes):
-        if magic_hex != ord_(data[i]):
+        if magic_hex != data[i]:
```

Comment:
it's fine without any `ord()`?

Reply:
Yes, the `ord_` compatibility function wasn't just like the regular `ord` in python3. You can try to run the test with `ord` if you want, but you will get an error:
```Traceback (most recent call last):
  File "/code/pygame/test/image_test.py", line 254, in test_save
    (1, fmt), (test_magic(handle, magic_hex[fmt.lower()]), fmt)
  File "/code/pygame/test/image_test.py", line 27, in test_magic
    if magic_hex != ord(data[i]):
TypeError: ord() expected string of length 1, but int found```